File: app/src/main/java/it/anyplace/syncbrowser/activities/MIVFilePickerActivity.java
package it.anyplace.syncbrowser.activities;

import com.nononsenseapps.filepicker.AbstractFilePickerActivity;
import com.nononsenseapps.filepicker.AbstractFilePickerFragment;

import java.io.File;

import it.anyplace.syncbrowser.fragments.MIVFilePickerFragment;

public class MIVFilePickerActivity extends AbstractFilePickerActivity<File> {

        @Override
        protected AbstractFilePickerFragment<File> getFragment(
                final String startPath, final int mode, final boolean allowMultiple,
                final boolean allowCreateDir) {
            // Only the fragment in this line needs to be changed
            AbstractFilePickerFragment<File> fragment = new MIVFilePickerFragment();
            fragment.setArgs(startPath, mode, allowMultiple, allowCreateDir);
            return fragment;
        }
}

File: app/src/main/kotlin/net/syncthing/lite/activities/MIVFilePickerActivity.kt
package net.syncthing.lite.activities

import com.nononsenseapps.filepicker.AbstractFilePickerActivity
import com.nononsenseapps.filepicker.AbstractFilePickerFragment

import net.syncthing.lite.fragments.MIVFilePickerFragment

import java.io.File

class MIVFilePickerActivity : AbstractFilePickerActivity<File>() {

    override fun getFragment(startPath: String, mode: Int, allowMultiple: Boolean,
            allowCreateDir: Boolean): AbstractFilePickerFragment<File> {
        // Only the fragment in this line needs to be changed
        val fragment = MIVFilePickerFragment()
        fragment.setArgs(startPath, mode, allowMultiple, allowCreateDir)
        return fragment
    }
}
